### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by natke**
Edit README for clarity and add latest functionality

### CODE REVIEW

**Review by xadupre — APPROVED** [review]

**Comment by GeorgeS2019** [comment]
@natke
I raise an issue to request for[ a better documentation](Documentation on how to coordinate the external Onnxruntime-Extension) how the onnxruntime-extension is maintained with respect to the onnxruntime. 

It seems to me here is the repository to coordinate the development of contributed operators to be eventually integrated and embedded into the onnxruntime

**Comment by snnn** [comment]
The pipeline failed because this PR is from a fork. I can modify the pipeline's setting to allow such build jobs pass, but it also requires code reviewers carefully review external PRs before running build pipelines for them.

**Comment by natke** [comment]
What do you think is best @snnn?

**Comment by GeorgeS2019** [comment]
Feedback

 > A library of common pre and post processing operators for vision, text, and nlp models 

what are the [operators ](https://github.com/microsoft/onnxruntime/blob/master/docs/ContribOperators.md)for vision? Text? and NLP? Which are typically used for pre and for post processing


Vision
- com.microsoft.CropAndResize

NLP
- com.microsoft.Tokenizer
- com.microsoft.BeamSearch

Which are pre?
- com.microsoft.Tokenizer

Which are post?

**Comment by natke** [comment]
@GeorgeS2019 Thank you for the questions. I have started to update the list of operators in this PR. See https://github.com/microsoft/onnxruntime-extensions/blob/6654b82ba575cac832ac208d5f8b9b06f65e97be/docs/custom_ops.md

File: README.md [hunk 1]
```diff
@@ -1,88 +1,163 @@
...
-next_id = numpy.argmax(output[:, :, -1, :], axis=-1)
-print(input_text[0] + decode(next_id).item())
+from pathlib import Path
+import torch
+from transformers import AutoTokenizer
+import onnx
+from onnxruntime_extensions import pnp
+
+# get an onnx model by converting HuggingFace pretrained model
+bert_model_name = "distilbert-base-uncased-finetuned-sst-2-english"
```
[the remark this answers is not in the document]
Comment (natke) [reply README.md]: I added the code to export the model from HuggingFace

File: README.md [hunk 2]
```diff
@@ -1,88 +1,197 @@
...
+## Quick Start
+
+### Installation
+
+#### Install from PyPI
 
-# Quick Start
-### **Installation**
-The package can be installed by standard pythonic way, ```pip install onnxruntime-extensions```.
+The latest release of onnxruntime-extensions is: 0.4.2.
```
[the remark this answers is not in the document]
Comment (natke) [reply README.md]: Good idea. Removed

**Review by wenbingl — APPROVED** [review]

**Pull request merged**